common.py

```diff
@@ -0,0 +1,32 @@
+import argparse
+
+def parse(line, is_labeled=True):
+    tokens = line.split(',')
+    features = [tokens[3], tokens[4], tokens[5], tokens[13], tokens[14], tokens[15], tokens[16], 
+        tokens[17], tokens[20], tokens[21], tokens[22]]
+    features = [float (f) for f in features]
+    if is_labeled == False:
+    	return (features, None)
+    else: 
+    	label = int (tokens[23])
```

Comment:
You can go even further and write `return (features, int(tokens[23] if is_labeled else None)`.